app/Console/Commands/UpdateSignupAndPostField.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+use Rogue\Managers\PostManager;
+use Rogue\Managers\SignupManager;
+
+class UpdateSignupAndPostField extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:updatefield {--target= : The name of the field to update} {--targetOldValue= : The value to search the target with} {--targetNewValue= : The value to update the target with}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change all target old values to target new value in the signups and posts tables';
+
+    /**
+     * The signup manager instance.
+     *
+     * @var Rogue\Managers\SignupManager
+     */
+    protected $signups;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(SignupManager $signups, PostManager $posts)
+    {
+        parent::__construct();
+
+        $this->signups = $signups;
+        $this->posts = $posts;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Start updating signups
+        info('rogue:updatefield: Starting to update signups!');
+
+        if (! $this->option('target')) {
+            $this->error('No target field specified.');
+
+            return;
+        }
+
+        if (! $this->option('targetOldValue')) {
+            $this->error('No target old value specified.');
+
+            return;
+        }
+
+        if (! $this->option('targetNewValue')) {
+            $this->error('No target new value specified.');
+
+            return;
+        }
```

Comment:
If these are all required I wonder if they should be regular arguments instead of options so that we don't have to do these checks ourselves?

Reply:
ah I think you're right! I will update that now :)